src/sidebar/store/modules/frames.ts

```diff
@@ -1,8 +1,4 @@
-import {
-  createSelector,
-  createSelectorCreator,
-  defaultMemoize,
-} from 'reselect';
+import { createSelector, createSelectorCreator, lruMemoize } from 'reselect';
```

Comment:
`defaultMemoize` has been renamed to `lruMemoize`, as it's no longer the default: https://github.com/reduxjs/reselect/releases/tag/v5.0.1